README.md

```diff
@@ -38,14 +38,21 @@ can be handled.
 
 It has been tested with Ruby 2.6.
 
+### Usage / Updating
+
+```fish
+./gen_completions.rb docker > completions/docker.fish
+./gen_completions.rb docker-compose > completions/docker-compose.fish
+```
```

Comment:
I haven’t written that part of the README because my plan is to change how this is done so that it’s _just_ `ruby ./gen_completions` and it runs both `docker` and `docker-compose` at once and writes the files to a single place. I haven’t had a chance to finish this part.